Public/Src/Engine/Scheduler/Graph/PipGraphFragmentManager.cs

```diff
@@ -20,15 +22,24 @@ namespace BuildXL.Scheduler.Graph
     /// </summary>
     public class PipGraphFragmentManager : IPipGraphFragmentManager

```

Comment:
Ditto, can fragment management move to BuildXl.Pips assemlby?

Reply:
This class adds deserialized pips to pip graph. So, I think it's natural to place it in the PipGraph assembly.

---
In reply to: [313653201](https://github.com/microsoft/BuildXL/pull/730#discussion_r313653201) [](ancestors = 313653201)

Reply:
note we don't have a pipgraph assembly, this is BuildXL.Schedule.dll :)

---
In reply to: [313718382](https://github.com/microsoft/BuildXL/pull/730#discussion_r313718382) [](ancestors = 313718382,313653201)